applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -162,6 +162,50 @@ For an example of bootloader Kconfig configuration file defined by the applicati
   Both mentioned firmware upgrade methods are not used simultaneously by any of the configurations.
   For example, the ``nrf52840dk/nrf52840`` board in ``prj_mcuboot_smp.conf`` uses only the background DFU and does not enable the serial recovery feature.
 
+.. _nrf_desktop_suit:
+
+Software Update for Internet of Things (SUIT)
+=============================================
+
+Software Update for Internet of Things (SUIT) is a metadata format and procedure for performing Device Firmware Update (DFU).
+It allows you to update and boot multiple CPU cores of the supported System on Chip (SoC).
+The SUIT DFU is currently available only for the nRF54H20 SoC.
+The application stores an update image in the dedicated DFU partition and triggers firmware update using the SUIT API.
+The firmware update affects the firmware of both the Application and Radio cores.
+The update candidates are moved directly to the corresponding image slots.
+
+The SUIT can be used for the :ref:`background DFU <nrf_desktop_bootloader_background_dfu>` through:
+
+   * The :ref:`nrf_desktop_config_channel` and :ref:`nrf_desktop_dfu`.
+   * The :ref:`nrf_desktop_dfu_mcumgr_suit`.
+
+For an introduction to SUIT and more detailed information, see the :ref:`ug_nrf54h20_suit_dfu`.
+
+Configuring SUIT
+----------------
+
+To enable SUIT on the supported target, enable the :kconfig:option:`CONFIG_SUIT` Kconfig option in the application configuration.
+This option enables services on application core that communicates with the Secure Domain on which the SUIT is executed.
+You can also enable :kconfig:option:`CONFIG_SUIT_DFU_CANDIDATE_PROCESSING_MINIMAL`, because application core only stores the SUIT envelope to a dedicated partition and then it passes control to the Secure Domain.
+
+SUIT also has the following options in the sysbuild configuration:
+
+   * :kconfig:option:`SB_CONFIG_SUIT_ENVELOPE` - Required to create the SUIT envelope.
+     Turned on by default on ``nrf54h20dk`` board.
+     The envelope is used directly as DFU update file by DFU tools.
+   * :kconfig:option:`SB_CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM` - The nRF Desktop application uses the sequence number to define an application version.
+
+.. note::
+   The :kconfig:option:`SB_CONFIG_SUIT_ENVELOPE_SIGN` is disabled, so the generated SUIT envelope is not signed.
+
+On ``nrf54h20dk`` board the dedicated DFU partition called ``dfu_partition`` is defined by default in the DTS.
+The partition is used to store the incoming SUIT envelope with update candidate.
+For details about defining a memory layout for your target, see the :ref:`nrf_desktop_memory_layout_dts` section.
```

Comment:
The linked information may actually be a bit misleading for nRF54H (memory configuration for this multi-core SoC is more complicated). Do we have a guide for nRF54H memory configuration in the NCS? If so I would link there instead. Otherwise we need to either write a short guide in scope of nRF Desktop doc or inform that we rely on default memory map and not to link DTS configuration section here.